blackcat/blackcat/storysharing/views.py

```diff
@@ -37,10 +41,75 @@ def send_email_to_active_writers(self, story, update):
             )
 
 
+class PrintableStoryView(DetailView, EmailActiveWritersMixin):
+    template_name = 'storysharing/printable_story.html'
+    model = Story
+    form_name = StorySettingsForm
+
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        if slugify(self.object.title) != kwargs['title']:
+            return HttpResponseRedirect(reverse('index'))
+        context = self.get_context_data(object=self.object)
+        context['form'] = self.form_name(
+            initial={
+                "shareable": self.object.shareable,
+                "public": self.object.public
+            })
+        context['is_writer'] = request.user.username in [
+            x['username'] for x in self.object.writers.values()]
+        return self.render_to_response(context)
+
+    def post(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        context = self.get_context_data(object=self.object)
+        context['is_writer'] = request.user.username in [
+            x['username'] for x in self.object.writers.values()]
```

Comment:
This could also be done as:
`context['is_writer'] = request.user in self.objects.writers.all()`